pkg/make_rpm.py

```diff
@@ -221,11 +223,17 @@ def CallRpmBuild(self, dirname):
         self.spec_file,
     ]
     os.environ['RPM_BUILD_ROOT'] = buildroot
+    env = {
```

Comment:
This mix of os.environ setting and env is strange. It never should have been that way in the first place.  It is not even clear how things worked the way they are, because using env at all would have suppressed RPM_BUILD_ROOT from being passed (at least in Python 3.7)  If we put everything in env it may be easier to understand.

Reply:
I made the suggested change in the most recent commit, and also addressed the test failures.